### PR TITLE
Created Version Changes Listing

### LINKED ISSUE #3462: What's new dialog upon 1st launch after app is update
Show a list of changes. Textual, for example the following, but probably shorter:

> * Siri shortcut to show your wallet QR code, available in Siri suggestions and Spotlight
> * Custom RPC servers can now be modified
> * Improve ENS avatar support for SVG
> * Many UI tweaks
> * Bug fixes

Users tend not to read release notes especially if they have auto-update enabled. Also opens up the opportunity to use the same UI to highlight specific 1-2 features at selected times.

Might tweak this format to just be about a single feature with a lengthier feature in the future, but let's roll something out first.

### PR DESCRIPTION
**Pull request by eviltofu**
Closes #3462

## Testing
1. Press the camera button on the Wallet screen
2. **Expects** What's new screen to pop up.

### CODE REVIEW

**Comment by hboon** [comment]
@eviltofu Thanks. Will check. Would you help make a single screenshot in such PRs? Helps a bit.

**Comment by eviltofu** [comment]
@hboon here are the screen shots

Simulator Screen Shot - iPhone 13 - 2021-11-30 at 11 31 14
Simulator Screen Shot - iPhone 13 - 2021-11-30 at 11 31 21

**Comment by eviltofu** [comment]
Note: Tested with Right to Left and Left to Right script.

**Comment by hboon** [comment]
@eviltofu looks right to me. I'll follow up and stuff things inside. Thanks!

File: AlphaWallet/Tokens/Views/VersionChangeViews.swift [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            self.tickViewImageView.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 16.0),
+            self.tickViewImageView.trailingAnchor.constraint(equalTo: self.entryLabel.leadingAnchor, constant: -8.0),
+            self.entryLabel.topAnchor.constraint(equalTo: self.topAnchor),
+            self.entryLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -48.0),
+            self.entryLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -8.0)
+        ])
+    }
+}
+
+//class VersionChangeSpaceView: UIView {
```
Comment (hboon) [inline AlphaWallet/Tokens/Views/VersionChangeViews.swift]: Remember to remove commented out code.

**Comment by hboon** [comment]
It's great that there's a clearer separation of models and view models. Raised the average :)

Took the liberty to make a few changes:

* Rename occurrences of "VersionChange" to "WhatsNew" in type names be be more intention revealing. We wouldn't necessarily show this only for changes introduced in new versions nor list all the changes in each new version
* Rename occurrences of "version" to names to "title". I can't think of a better name, but want to move away from it being about version only
* Add the ability to show/hide the ticks
* Wrap the "version" label
* Move the files into a `WhatsNew` folder. Not too sure about it ending up with 1 file in each subfolder (eg. 1 file in `Views/`). But it's a least consistent

Have also rebased master.

@eviltofu Can you review it and let me know if it looks good?

**Comment by hboon** [comment]
Forgot to add. The changes supports this too:

Screenshot 2021-12-01 at 12 47 01 PM

**Comment by eviltofu** [comment]
@hboon code looks ok but I am not able to test it.

**Comment by hboon** [comment]
Force pushed after rebase master.

**Pull request merged**